src/control/lib/support/log.go

```diff
@@ -48,15 +50,31 @@ type CollectLogSubCmd struct {
 	TargetFolder string `short:"t" long:"target-folder" description:"Target Folder location where log will be copied"`
 	Archive      bool   `short:"z" long:"archive" description:"Archive the log/config files"`
 	ExtraLogsDir string `short:"c" long:"extra-logs-dir" description:"Collect the Logs from given directory"`
+	LogStartDate string `short:"D" long:"start-date" description:"Specify the start date, the day from log will be collected, Format: MM-DD"`
+	LogEndDate   string `short:"F" long:"end-date" description:"Specify the end date, the day till the log will be collected, Format: MM-DD"`
+	LogStartTime string `short:"S" long:"log-start-time" description:"Specify the log collection start time, Format: HH:MM:SS"`
+	LogEndTime   string `short:"E" long:"log-end-time" description:"Specify the log collection end time, Format: HH:MM:SS"`
 }
 
+type LogTypeSubCmd struct {
+	LogType string `short:"e" long:"log-type" description:"collect specific logs only admin,control,server and ignore everything else"`
```

Comment:
Would be helpful to indicate the format for combining these options. In the tests it seems to be a `|` not a `,`.

Reply:
I will add the combination option in help.